cmd/agent/dist/conf.d/disk.d/conf.yaml.default

```diff
@@ -1,53 +1,162 @@
+## This file is overwritten upon Agent upgrade.
+## To make modifications to the check configuration, copy this file
+## to `disk.yaml` and make your changes on that file.
+
 init_config:
 
 instances:
-  # The use_mount parameter will instruct the check to collect disk
-  # and fs metrics using mount points instead of volumes
+
+    ## @param use_mount - boolean - required
+    ## Instruct the check to collect using mount points instead of volumes.
+    #
   - use_mount: false
-    # The (optional) excluded_filesystems parameter will instruct the check to
-    # ignore disks using these filesystems. Note: On some linux distributions,
-    # rootfs will be found and tagged as a device, add rootfs here to exclude.
+
+    ## @param excluded_filesystems - list of strings - optional - default: tmpfs
+    ## This parameters instructs the check to ignore disks using these filesystems.
+    ## Note: On some linux distributions, rootfs is found and tagged as a device.
+    ## Add rootfs here to exclude.
+    #
     # excluded_filesystems:
     #   - tmpfs
 
-    # The (optional) excluded_disks parameter will instruct the check to
-    # ignore this list of disks.
-    # Linux example:
+    ## @param excluded_disks - list of strings - optional
+    ##  The `excluded_disks` parameter instructs the check to
+    ## ignore this list of disks.
+    #
     # excluded_disks:
     #   - /dev/sda1
     #   - /dev/sda2
+
+    ## @param excluded_disk_re - string - optional
+    ## The `excluded_disk_re` parameter instructs the check to
+    ## ignore all disks matching this regex.
     #
-    # The (optional) excluded_disk_re parameter will instruct the check to
-    # ignore all disks matching this regex.
     # excluded_disk_re: /dev/sde.*
+
+    ## @param excluded_mountpoint_re - string - optional
+    ## The `excluded_mountpoint_re` parameter instructs the check to
+    ## ignore all mountpoints matching this regex.
     #
-    # The (optional) tag_by_filesystem parameter will instruct the check to
-    # tag all disks with their filesystem (for ex: filesystem:nfs).
-    # tag_by_filesystem: false
-    #
-    # The (optional) excluded_mountpoint_re parameter will instruct the check to
-    # ignore all mountpoints matching this regex.
-    # excluded_mountpoint_re: /mnt/somebody-elses-problem.*
+    # excluded_mountpoint_re: <MOUNT_POINT_REGEX>
+
+    ## @param all_partitions - boolean - optional
+    ## Instruct the check to collect from partitions even without device names.
+    ## Setting `use_mount` to true is strongly recommended in this case.
     #
-    # The (optional) all_partitions parameter will instruct the check to
-    # get metrics for all partitions. use_mount should be set to yes (to avoid
-    # collecting empty device names) when using this option.
     # all_partitions: false
+
+    ## @param tags - list of key:value string - optional
```

Comment:
```suggestion
    ## @param tags - list of key:value strings - optional
```

Reply:
The rest of the PR uses `list of key:value elements`, which one should be prefered?